docs/docs-content/clusters/data-center/nutanix/create-manage-nutanix-cluster.md

```diff
@@ -39,80 +39,92 @@ section guides you in creating a Kubernetes cluster in a Nutanix cloud managed b
+7.  Review profile layers, leaving the OS and Kubernetes packs empty, and customize parameters as desired in the YAML
+    files that display when you select the network and storage layers. Click **Next** when you are done.
 
-| **Variable**                   | **Description**                                                                              |
-| ------------------------------ | -------------------------------------------------------------------------------------------- |
-| `${CLUSTER_NAME}`              | The name of the Nutanix workload cluster. Use the same cluster name you specified in step 5. |
-| `${CONTROL_PLANE_ENDPOINT_IP}` | The Kubernetes API IP endpoint for the cluster you are creating.                             |
-| `${NUTANIX_ENDPOINT}`          | The Nutanix Prism Central IP address.                                                        |
+8.  In the **Cluster Configuration Macros** pane, enter values that apply to your Nutanix cloud environment and, if
+    necessary, adjust the **Cluster configuration** YAML file to configure your cluster. Click **Next** when you are
+    ready to proceed.
 
-:::warning
+    :::info
 
-The following applies when replacing variables within curly braces in the YAML configuration files.
+    The inactive fields are auto-populated from your Nutanix cloud account configuration. When entering other macros,
+    make sure to verify the default values.
 
-    - All the variables must be resolved or have a default value.
+    :::
 
-    - Verify default values such as the port.
+    | **Field**                                      | **Description**                                                                                                                                                                                                                                                                                        |
+    | ---------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+    | **NUTANIX_ADDITIONAL_TRUST_BUNDLE (Optional)** | An environment variable that allows CAPX to verify certificates that are not issued by a publicly trusted certificate authority. For more information, refer to the [Nutanix Certificate Trust](https://opendocs.nutanix.com/capx/v1.2.x/pc_certificates/#configure-an-additional-trust-bundle) guide. |
+    | **CONTROL_PLANE_ENDPOINT_IP**                  | The host IP and port of the CAPX Kubernetes cluster.                                                                                                                                                                                                                                                   |
```

Comment:
```suggestion
    | **CONTROL_PLANE_ENDPOINT_IP**                  | The host IP of the CAPX Kubernetes cluster.                                                                                                                                                                                                                                                   |
```

Reply:
Maybe?

Reply:
Yup, you're right! I took this from [controlPlaneEndpoint](https://opendocs.nutanix.com/capx/v1.0.x/types/nutanix_cluster/) but this one is specifically for the IP. Thanks Caro :)